packages/userprofile/README.md

```diff
@@ -13,40 +16,87 @@ The UserProfile extension has the following peer dependency, which must be insta
 
 See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page 
 
+Install the `@adobe/react-native-aepuserprofile` package:
+
+```bash
+npm install @adobe/react-native-aepuserprofile
+```
+
 ## Usage
 
 ### Initializing and registering the extension
 
 Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
 
-### [User Profile](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/profile)
-
 #### Importing the extension:
 ```javascript
 import {UserProfile} from '@adobe/react-native-aepuserprofile';
 ```
 
-#### Getting the extension version:
+## API reference
+
+- #### extensionVersion
```

Comment:
Use ### extensionVersion for consistency with other extensions?
Same as other API in user profile doc.